runelite-client/src/main/java/net/runelite/client/plugins/itemprices/ItemPricesOverlay.java

```diff
@@ -267,7 +271,7 @@ private String stackValueText(int qty, int gePrice, int haValue, int haProfit)
 
 			itemStringBuilder.append("</br>");
 			itemStringBuilder.append("HA Profit: ")
-				.append(ColorUtil.wrapWithColorTag(String.valueOf(haProfit * qty), haColor))
+				.append(ColorUtil.wrapWithColorTag(String.valueOf(haStackPrice), haColor))
```

Comment:
You seem to be replacing the total alch profit by the total alch price here.

Reply:
You're right, I can fix that